src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -15,6 +15,8 @@ public class Arm extends SubsystemBase {
       new CANSparkMax(CanId.shoulderNeo2, MotorType.kBrushless);
   private final CANSparkMax elbowNeo = new CANSparkMax(CanId.elbowNeo, MotorType.kBrushless);
   private final WPI_TalonSRX leftLead = new WPI_TalonSRX(CanId.turret);
+  private final double As = 0.0;
```

Comment:
These should be the Lp/Lf (length proximal and forearm variables) and the method parameters As Ae. Also maybe rename those theta_shoulder or smth so it's a bit more readable.

Lf and Lp should also go in Constants.java, but all the arm/gripper constants need this, so I'll open an issue for all of them to be resolved in a separate PR.

Good work :+1: